utils/src/main/java/io/neow3j/contract/ScriptBuilder.java

```diff
@@ -132,6 +132,12 @@ public ScriptBuilder pushParam(ContractParameter param) {
             case ARRAY:
                 pushArray((ContractParameter[]) value);
                 break;
+            case ANY:
+                if (value == null) {
+                    opCode(OpCode.PUSHNULL);
+                }
+                // TODO: 23.12.20 Michael: Serialize ContractParameter Any
```

Comment:
Create an issue for the missing serialization and deserialization of the Any contract parameter and remove this TODO.

Reply:
I created an issue (#344) for that.